src/datatypes/numeric.js

```diff
@@ -24,7 +24,14 @@ function generateFunctions(bufferReader,bufferWriter,size)
     };
   };
   var writer=function(value, buffer, offset) {
-    buffer[bufferWriter](value, offset);
+    try {
```

Comment:
We could use [debug](https://github.com/visionmedia/debug) module to fix this. By making multiple debug modules for each part of the application, we could remove the spammy logging done.

It might also be a good idea to just remove the spam per-packet in general. I don't think it's actually useful, and can be turned back on when we need it ?